.build/.k.rev

```diff
@@ -1 +1 @@
-45820d1cd56ae7b3d4bc421134e884bbbd7b2f68
+2b51f3589e5922ed4f12784e982105249f623120
```

Comment:
https://github.com/kframework/k/compare/45820d1cd56ae7b3d4bc421134e884bbbd7b2f68...2b51f3589e5922ed4f12784e982105249f623120

Reply:
Here are the 2 commits related to [matching]:
https://github.com/kframework/k/commit/7e8f009af84746f838626a206391cbbe082d52d8
https://github.com/kframework/k/commit/2b51f3589e5922ed4f12784e982105249f623120

Reply:
three commits are non-logging one.
1. https://github.com/kframework/k/commit/0eece27d90865a9d726d864e1915ccd866c46a1b
2. https://github.com/kframework/k/commit/7e8f009af84746f838626a206391cbbe082d52d8
3. https://github.com/kframework/k/commit/2b51f3589e5922ed4f12784e982105249f623120

(1) and (2) look ok to me, but i need more time to think about (3).